### DISCUSSION TITLE
How to I add and/or modify a contact for an alias?

### DISCUSSION THREAD

**Discussion by meslogiciels** [simple-login/app#2263] [Q&A]
I am currently on the SimpleLogin Free plan. I understand that I am able to create up to 10 aliases under this plan. I currently have created 3 aliases.
Under each of my aliases, there is a 'Contacts' button. When I hoover over the button, the tooltip states that the purpose of the button is to add new contact and/or manage existing contact.
When I click on the 'Contacts' button, the only option that I have is to search for contacts. I don't see any option to add a new contact or manage an existing one.
I must be missing something. How to I add and/or modify a contact for an alias?

**Comment by nguyenkims** [comment]
Creating a contact requires a premium plan.
![Uploading image.png…]()
